source/reference/minio-mc/mc-idp-openid.rst

```diff
@@ -0,0 +1,309 @@
+.. _minio-mc-idp-openid:
+
+=================
+``mc idp openid``
+=================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc idp openid
+
+.. versionadded:: RELEASE.2023-05-26T23-31-54Z
```

Comment:
Maybe worth adding a note here that it replaces the `mc admin` command.

Reply:
Oh right, that would be a good thing to include. Good catch.